src/Classes/ServiceAPI/MyRadio_TracklistItem.php

```diff
@@ -404,7 +405,7 @@ public static function getIfAlbumArtistCompliant(MyRadio_Track $track, $include_
             }
         }
 
-        return $result[0] < 2;
+        return $result[0] = 0;
```

Comment:
<1 maybe? All I know is that it should only return a result if there are no matches within an hour.